scripts/license/LICENSE-template.md

```diff
@@ -0,0 +1,25 @@
+changequote("""","""")dnl # prevents m4 from being confused with backquotes by changing quotes to non-existent tokens
+<!-- This file is generated, edit LICENSE-header.md to update the license. -->
+## License
+
+Mapbox Maps for iOS version __MAPS_SDK_VERSION__
+Mapbox Maps iOS SDK
+
+Copyright &copy; 2021 - 2023 Mapbox, Inc. All rights reserved.
+
+The software and files in this repository (collectively, “Software”) are licensed under the Mapbox TOS for use only with the relevant Mapbox product(s) listed at www.mapbox.com/pricing. This license allows developers with a current active Mapbox account to use and modify the authorized portions of the Software as needed for use only with the relevant Mapbox product(s) through their Mapbox account in accordance with the Mapbox TOS.  This license terminates automatically if a developer no longer has a Mapbox account in good standing or breaches the Mapbox TOS. For the license terms, please see the Mapbox TOS at https://www.mapbox.com/legal/tos/ which incorporates the Mapbox Product Terms at www.mapbox.com/legal/service-terms.  If this Software is a SDK, modifications that change or interfere with marked portions of the code related to billing, accounting, or data collection are not authorized and the SDK sends limited de-identified location and usage data which is used in accordance with the Mapbox TOS. [Updated 2023-01]
+
+## Acknowledgements
+
+This application makes use of the following third party libraries:
+
+### turf-swift, https://github.com/mapbox/turf-swift
+
+```
+__TURF_LICENSE_CONTENT__
+```
+
+---
+
+__CORE_LICENSE__
+<!-- End of generated file. -->
```

Comment:
I find it a bad idea to include generation note in the public facing license (and we do include it in the binary form along with release available content). 
While including sub-licenses is fine, I suppose we have to receive legal approval for including extra data/metadata in the license content itself.

Reply:
Approved by @sbma44. I thought that adding this note would help to prevent situations when the generated file is modified(when license is updated) mistakenly instead of the template. However, if you think it's a bad idea I'm open to removing these comments.

Reply:
I suppose our CI process would prevent such situation, right? 
As Tom already approved it then it fine, but I personally would prefer a "clean" license version 🙂

Reply:
True that, license check would fail in this case, I'll remove the comments then